styleguide/_themes/derek/scss/patterns/lead.scss

```diff
@@ -1,18 +1,18 @@
-.lead-bodyWrapper {
-  margin-bottom: 14px;
```

Comment:
These aren't changes, just moved some things around to be "as they appear on the page" in order